src/v2/Components/Footer.tsx

```diff
@@ -297,9 +304,13 @@ const PolicyLinks = () => (
     </Link>
 
     <Link to="/conditions-of-sale">
-      <Text variant="caption" color="black60">
+      <Text variant="caption" color="black60" mr={1}>
         Conditions of Sale
       </Text>
     </Link>
+
+    <CCPAWrapper>
+      <CCPARequest />
+    </CCPAWrapper>
```

Comment:
Can we [add a test](src/v2/Components/__tests__/Footer.jest.tsx) to make sure that this `CCPARequest` exists? That way, in the future, if someone accidentally removes it (like what happened last month) it will set of a red flag.

Reply:
Added in a test for this!